packages/webview_flutter/webview_flutter_android/android/src/main/java/io/flutter/plugins/webviewflutter/WebViewHostApiImpl.java

```diff
@@ -77,7 +60,12 @@ public void setWebContentsDebuggingEnabled(boolean enabled) {
   }
 
   /** Implementation of {@link WebView} that can be used as a Flutter {@link PlatformView}s. */
+  @SuppressLint("ViewConstructor")
```

Comment:
Why doesn't this show up in our linting in CI? Is it something that we need to explicitly enable in lint options?

Reply:
It seems the flag at https://github.com/flutter/packages/blob/main/packages/webview_flutter/webview_flutter_android/example/android/build.gradle#L38 doesn't include this?

I had to update the plugin `build.gradle` to get the lint to cause the build to fail: https://github.com/flutter/packages/pull/3428/commits/42e45ff299f7527941fbe056623f760035bbb7d9

Is this a suitable solution? I would imagine this should be applied to all our plugins.

Reply:
> It seems the flag at https://github.com/flutter/packages/blob/main/packages/webview_flutter/webview_flutter_android/example/android/build.gradle#L38 doesn't include this?

That applies to `javac` lints, which is [a whole different thing](https://docs.oracle.com/javase/7/docs/technotes/tools/windows/javac.html#xlintwarnings)

This looks like the other linter (`./gradlew lint`) which we run via our `lint-android` tooling command. We should probably see if we can enable warnings as errors in general with that; I thought it would already fail on warnings but I guess not?